Libs/RaptureWebServlet/src/main/java/rapture/server/web/servlet/UserBoundedScriptPageServlet.java

```diff
@@ -193,6 +246,35 @@ protected void runScriptWithUser(String script, Map<String, Object> parameterMap
         resp.flushBuffer();
     }
 
+    /**
+     * Helper method to extract the URI from HttpServletRequest
+     * <p>
+     * @param  request  The actual script/endpoint to execute  
+     * @return          URI of script as a String e.g. script://<repo>/endpoint 
+     */
+    private String getScriptUrlFromHttpRequest(HttpServletRequest request){
+        String scriptURL = "";
+        
+        String path = request.getPathInfo();
```

Comment:
getPathInfo from HttpServletRequest can return null.  From the javadoc:

"This method returns null if there was no extra path information."

Might be good to check if its null before split()ing.